lxc/copy.go

```diff
@@ -372,11 +370,16 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 	progress.Done("")
 
 	if c.flagRefresh {
-		_, etag, err := dest.GetInstance(destName)
+		inst, etag, err := dest.GetInstance(destName)
 		if err != nil {
 			return fmt.Errorf("Failed to refresh target container '%s': %v", destName, err)
 		}
 
+		// Ensure we don't change the target's volatile.idmap.next value.
+		if inst.Config["volatile.idmap.next"] != "" {
+			writable.Config["volatile.idmap.next"] = inst.Config["volatile.idmap.next"]
+		}
```

Comment:
I think we want the event simpler `writable.Config["volatile.idmap.next"] = inst.Config["volatile.idmap.next"]` so that even if the value wasn't set locally and is now set remotely we don't go messing with it still.

Reply:
@stgraber have pushed that change.